ci/tasks/build-deployable.yml

```diff
@@ -16,6 +16,7 @@ params:
 
 inputs:
   - name: src
+  - name: wordlist-file
```

Comment:
I may be missing something, where is the wordlist-file coming from?

Reply:
https://github.com/alphagov/govwifi-concourse-deploy-pipeline/pull/3/files

Reply:
This is an input to the task from the s3 resource defined in alphagov/govwifi-concourse-deploy-pipeline#3

Reply:
Seen it. I suppose there is no way of testing till both pipelines are in master? I would like to see it running but since the pipeline is broken right now, I am happy to approve and roll back if needed.